src/commands/subcommands/misc/handleEcho.ts

```diff
@@ -0,0 +1,28 @@
+import { CommandHandler } from "../../../interfaces/commands/CommandHandler";
+import { errorEmbedGenerator } from "../../../modules/commands/errorEmbedGenerator";
+import { beccaErrorHandler } from "../../../utils/beccaErrorHandler";
+
+/**
+ * Echos what the user type.
+ */
+export const handleEcho: CommandHandler = async (Becca, interaction, t) => {
+  try {
+    interaction.ephemeral = true;
+    const input = interaction.options.getString("input") || "";
```

Comment:
```suggestion
    const input = interaction.options.getString("input", true);
```

It's a required input, so we call tell the `.getString()` method such and avoid needing a fallback.

Reply:
Right. Yeah it shouldn't need a fallback.